README.md

```diff
@@ -43,53 +43,132 @@ exports.passport = {
 ### Using github and twitter strategy
 
 ```js
-// app.js
+// config/config.default.js
+exports.passport = {
+  github: {
```

Comment:
That's a problem. Or should we separate `egg-passport` from `egg-passport-github`? 

https://github.com/eggjs/egg/issues/397

cc @fengmk2